### DISCUSSION TITLE
Just wanted to confirm whether below features available or not with MAC operating system.

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8212] [Q&A] [answered]
Flexible Database Management (If yes) How exactly it will work, I am not seeing much information for MAC
Encrypted Database (If yes) How exactly it will work, I am not seeing much information for MAC
TOTP option for 2-factor authentication (If yes) How exactly it will work, I am not seeing much information for MAC
Supports static keys (Yubikey,Onlykey etc) I believe this is supported with MAC
Password Generator & Passphrase Generator I believe this is supported with MAC
Browser Integration
OpenSSH Integration
Screenshot Security
Quick Unlock

If any other feature is important and not listed then please highlight.
If this is not the right place , Please help me with the correct forum.

**Comment by droidmonkey** [comment, marked as answer]
All features work on all platforms. Read our user guide for information.
